review_mozilla_bitbar_devicepool/projects.py

```diff
@@ -0,0 +1,213 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
It looks like `projects.py, files.py, runs.py, frameworks.py, taskcluster.py, devices.py and device_groups.py` are more or less just wrappers around bitbar's library.

At the very least it would be nice to group these into a subdirectory to differentiate them from the business logic of this module. Tbh, I'd probably just merge them all onto a single helper class, though I don't mind the module level function approach either.

Reply:
It was mostly a way to organize things during the rewrite. This will be handed off to Andrew soon and he/Releng can do what they want with it.